cmd/kubeadm/app/cmd/util/cmdutil.go

```diff
@@ -32,23 +32,29 @@ import (
 	kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
 )
 
-// SubCmdRunE returns a function that handles a case where a subcommand must be specified
+// SubCmdRun returns a function that handles a case where a subcommand must be specified
 // Without this callback, if a user runs just the command without a subcommand,
 // or with an invalid subcommand, cobra will print usage information, but still exit cleanly.
 // We want to return an error code in these cases so that the
 // user knows that their command was invalid.
-func SubCmdRunE(name string) func(*cobra.Command, []string) error {
-	return func(_ *cobra.Command, args []string) error {
-		if len(args) < 1 {
-			return errors.Errorf("missing subcommand; %q is not meant to be run on its own", name)
+func SubCmdRun() func(*cobra.Command, []string) {
```

Comment:
Is it better to keep `SubCmdRunE` here? 
It seems that when this function is called, it should always exit with error.

> // We want to return an error code in these cases so that the
// user knows that their command was invalid.

Reply:
Can we just change the error message content here to contain the help info?